firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -94,12 +97,12 @@ public Thread newThread(Runnable r) {
       @Nullable HeartBeatInfo heartbeatInfo) {
     this(
         new ThreadPoolExecutor(
-            /* corePoolSize= */ 0,
-            /* maximumPoolSize= */ 1,
-            /* keepAliveTime= */ 30L,
+            /* corePoolSize= */ CORE_POOL_SIZE,
+            /* maximumPoolSize= */ MAXIMUM_POOL_SIZE,
+            /* keepAliveTime= */ KEEP_ALIVE_TIME_IN_SECONDS,
             TimeUnit.SECONDS,
             /* workQueue= */ new LinkedBlockingQueue<>(),
-            /* threadFactory= */ threadFactory),
+            /* threadFactory= */ THREAD_FACTORY),
```

Comment:
No comments needed since you are using constants that explain the purpose of the parameters.